Procfile

```diff
@@ -1 +1,2 @@
-web: node lib/main production
+web:                     node lib/main server
+expireCacheBusters: node lib/main expire-cache-busters
```

Comment:
Obivously, this means that someone spams the service and purges 1000 caches, then previous requests won't have effect...
But if there is so many it won't work now because you don't handle `continuationToken`... Anyways, if suddenly all the workers have to do paging I suspect it won't be performing well anymore either..

If we keep a single entity for each workerType with a list of things to purge. Then if more than if the entry removed when a new entry is added is less than 2 hours old we return 4xx indicating that the server can't handle more cache purges for the time being...

That way we'll only susceptible to denial of service attacks.. and only from privileged people..